controls/iam.sp

```diff
@@ -0,0 +1,26 @@
+locals {
+  iam_compliance_common_tags = merge(local.terraform_azure_compliance_common_tags, {
+    service = "Azure/IAM"
+  })
+}
+
+benchmark "iam" {
+  title       = "IAM"
+  description = "This benchmark provides a set of controls that detect Terraform Azure IAM resources deviating from security best practices."
+
+  children = [
+    control.iam_no_custom_subscription_owner_roles_created
+  ]
+
+  tags = merge(local.iam_compliance_common_tags, {
+    type = "Benchmark"
+  })
+}
+
+control "iam_no_custom_subscription_owner_roles_created" {
+  title       = "Ensure That No Custom Subscription Administrator Roles Exist"
```

Comment:
I think instead of `No custom subscription administrator roles should exist`, `Custom subscription administrator roles should not exist` would be more appropriate if you want to avoid the usage of the word `Ensure`